src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,38 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+
+export const Tabs = ({ tabs, selectedTabId, onTabSelected }) => {
+  const tab = tabs.find(tabsItem => selectedTabId === tabsItem.id) || tabs[0];
+  const eventHandler = (item) => {
```

Comment:
```suggestion
  const handleClick = (item) => {
```
we usually name it starting with `handle`